lib/libc/minimal/include/sys/types.h

```diff
@@ -28,7 +28,7 @@ typedef int off_t;
 typedef int off_t;
 #elif defined(__NIOS2__)
 typedef int off_t;
-#elif defined(__riscv__)
+#elif defined(__riscv)
```

Comment:
is this backward compatible with older GCC?

Reply:
yes